lib/Swagger/composer.json

```diff
@@ -17,15 +17,15 @@
         }
```

Comment:
************* START GENERATED CODE *************

The code from here up to the "END GENERATED CODE" comment below is all generated via swagger definition, feel free to skip down.  If you decide to review it understand that the only way to change this code is to update the swagger definition defined in the `user-preference` service.